### ISSUE TITLE
Allow disabling user initiated organizational invites when SSO is required

### ISSUE THREAD

**Issue by io41** [getsentry/sentry#62830]
### Problem Statement

When SSO is explicitly required for user creation and authentication, no invites to the organization should be sent, nor should admins need to approve, reject or have to decide to ignore invitations.

Unfortunately it's currently not possible to disable and prevent user's sending such invitations to the organization to new users.

This generates unnecessary notifications, including administrative ones asking if the invitation should be approved or not. In the age of information overload, each unnecessary context switch contributes to alert-fatigue and reduces productivity.

Invites to Sentry Team's are unaffected.

### Product Area

Unknown

┆Issue is synchronized with this [Jira Improvement](https://getsentry.atlassian.net/browse/FEEDBACK-2065) by [Unito](https://www.unito.io)

**Comment by azaslavsky** [comment]
Just to be clear: the reason why invites should be disallowed when SSO is enabled is because the SSO provider should manage access? So instead of receiving an invite from an existing user via the usual email mechanism, users are added via what other means?

**Comment by io41** [comment]
@azaslavsky yes, at least in the cases that SSO is setup with auto provisioning, which means users are added simply by signing in.

**Comment by azaslavsky** [comment]
Ok, this is more of a mainline product feature, so transferring to that repo.

**Comment by leedongwei** [comment]
Duplicated by https://github.com/getsentry/sentry/issues/56889. 

Fix planned https://github.com/getsentry/sentry/pull/73521

**Comment by sentaur-athena** [comment]
Closing since the fix PR is merged.
